psi4/src/psi4/libscf_solver/uhf.cc

```diff
@@ -287,15 +289,31 @@ void UHF::form_C(double shift) {
         shifted_F->add(Fb_);
         diagonalize_F(shifted_F, Cb_, epsilon_b_);
     }
-    if (options_.get_bool("GUESS_MIX") && (iteration_ == 0)) {
-        if (Ca_->nirrep() == 1) {
-            outfile->Printf("  Mixing alpha HOMO/LUMO orbitals (%d,%d)\n\n", nalpha_, nalpha_ + 1);
-            Ca_->rotate_columns(0, nalpha_ - 1, nalpha_, pc_pi * 0.25);
-            Cb_->rotate_columns(0, nbeta_ - 1, nbeta_, -pc_pi * 0.25);
-        } else {
+    if (options_.get_bool("GUESS_MIX") && !mix_performed_) {
+        if (Ca_->nirrep() != 1) {
+            outfile->Printf("  Mixing alpha HOMO/LUMO orbitals (%d,%d)\n", nalpha_, nalpha_ + 1);
             throw InputException("Warning: cannot mix alpha HOMO/LUMO orbitals. Run in C1 symmetry.",
                                  "to 'symmetry c1'", __FILE__, __LINE__);
         }
+
+        // SAD doesn't have orbitals in iteration 0, other guesses do
+        bool have_orbitals = !sad_ || (sad_ && iteration_ > 0);
```

Comment:
In my ideal world, we could just check if `Ca_` was `nullptr`, but getting Psi to such a state is beyond-the-scope of this PR.

Reply:
Yup...